tests/signal/test_signal_service.py

```diff
@@ -67,31 +66,26 @@ def test_filter_actions_deduplicate(session, signal, project):
         project=project,
     )
     session.add(entity_type)
+    signal.entity_types.append(entity_type)
 
-    entity = Entity(name="test", description="test", value="foo", entity_type=entity_type)
     session.add(entity)
 
     # create instance
     signal_instance_1 = SignalInstance(
         raw=json.dumps({"id": "foo"}), project=project, signal=signal, entities=[entity]
     )
     session.add(signal_instance_1)
-
     signal_instance_2 = SignalInstance(
         raw=json.dumps({"id": "foo"}), project=project, signal=signal, entities=[entity]
     )
     session.add(signal_instance_2)
-    signal.entity_types.append(entity_type)
-
     session.commit()
 
     # create deduplicate signal filter
     signal_filter = SignalFilter(
         name="test",
         description="test",
-        expression=[
-            {"or": [{"model": "EntityType", "field": "id", "op": "==", "value": entity_type.id}]}
-        ],
+        expression=[{"or": [{"model": "Entity", "field": "id", "op": "==", "value": entity.id}]}],
```

Comment:
Changed to resolve [test failure](https://github.com/Netflix/dispatch/actions/runs/4624040233/jobs/8178465630?pr=3228)

This passes.

cc @kevgliss